client/src/components/UserShowPage.js

```diff
@@ -2,110 +2,119 @@ import React, { useState } from 'react';
 import Dropzone from "react-dropzone"
 
 const UserShowPage = (props) => {
-    const { id } = props.match.params
-    const currentUser = props.currentUser
-    const [user, setUser] = useState({
-      id: '', email: '', createdAt: '', updatedAt: '', username: '', profileImage: ''
-    })
-    const [newProfileImage, setNewProfileImage] = useState({
-      image: {}
-    })
+  const { id } = props.match.params
+  const currentUser = props.currentUser
+
+  const [user, setUser] = useState({
+    id: '', email: '', createdAt: '', updatedAt: '', username: '', profileImage: ''
+  })
```

Comment:
probably dont need all of these properties on this default state, and it looks a bit wild. remove